util/library_loader.cc

```diff
@@ -54,9 +54,9 @@ size_t LibraryLoader::GetEntryPoints(std::map<std::string, void*>& functions) {
 
 UnixLibCrypto::UnixLibCrypto() : LibraryLoader(crypto_lib_name_) {
   if (is_valid_) {
-    // size of map minus two since _new/_create and _free/_destroy
-    //  only resolve one of the two.
-    is_valid_ = ((functions_.size() - 2) == GetEntryPoints(functions_));
+    // size of map minus three since _new/_create, _free/_destroy
```

Comment:
which version of OpenSSL does `_create` and `_destroy` comes from? sorry I didn't find the reference.